markdown/3.0/en/ff-navigation.md

```diff
@@ -5,14 +5,14 @@ To get a simple `ff-navigation` started, you only need to use the following code
 ```html
 <ff-navigation >
     <ff-navigation-item>
-        <div slot="name">{{name}}</div>
+        <div data-content="item">{{name}}</div>
     </ff-navigation-item>
 </ff-navigation>
 ```
 
-You need to have at least one `ff-navigation-item` defined. The `ff-navigation-item` is used as a template for all the elements of the Navigation. We use the term 'elements' to describe a single navigation category in a tree-like structure.
+You need to define least one `ff-navigation-item`. The `ff-navigation-item` is used as a template for all the elements of the Navigation. We use the term 'elements' to describe a single navigation category in a tree-like structure.
 
-In this `ff-navigation-item` you have access to the element's values. Such as `'name'` and `'recordCount'`. We define a `div` for the slot 'content' in which you define how each element of the navigation should be displayed. You can add a css class on this element:
+In scope of `ff-navigation-item` you have access to the navigation element's values, such as `name` and `recordCount`. We define a `div` for the item template in which you define how each element of the navigation should be displayed. You can add a css class on this element:
```

Comment:
```suggestion
Within `ff-navigation-item` you have access to the navigation element's values, such as `name` and `recordCount`. We define a `div` for the item template in which you define how each element of the navigation should be displayed. You can add a css class on this element:
```

Reply:
Within is related to time and distance at first. It means 'inside' as well. However, opening the sentence it doesn't look well. Scope is often used in IT, so in my opinion it fits in better our case.